DictionaryHomework9/Program.cs

```diff
@@ -0,0 +1,122 @@
+﻿using CityClass;
+
+namespace DictionaryHomework9
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            // Task 1
+            AddValuesToDictAndOutputThem();
+
+            // Task 2
+            List<string> listOfString = new List<string> { "Bear", "Wolf", "Rabbit", "Elephant", "Snake", "Frog", "Dog", "Cat", "Bat", "Lemur"};
+            List<int> listOfInts = new List<int> { 3, 7, 1, 9, 99, 51, 2, 90, 67, 77 };
+            var newDict = SortAndUnionGivenListsToDict(listOfString, listOfInts);
+            OutputDictionaryKeysValuesTask2(newDict);
+
+            // Task 3
+            City newYork = new City(8468000, 302.6);
+            City oslo = new City(634293, 175.3);
+            City copenhagen = new City(602481, 69.42);
+            City vienna = new City(1897000, 160.1);
+            City brno = new City(379526, 88.87);
+            Dictionary<string, City> citiesDictionary= new Dictionary<string, City>
+            {
+                { "New York", newYork },
+                { "Oslo", oslo },
+                { "Copenhagen", copenhagen },
+                { "Vienna", vienna },
+                { "Brno", brno }
+            };
+            Console.WriteLine("Sorted City Dictionary by Square:");
+            var sortedCitiesDictBySquare = SortCityDictBySquare(citiesDictionary);
+            OutputDictionaryKeysValuesTask3(sortedCitiesDictBySquare);
+            Console.WriteLine("Sorted City Dictionary by Population Desc:");
+            var sortedCitiesDictByPopulation = SortDescCityDictByPopulation(citiesDictionary);
+            OutputDictionaryKeysValuesTask3(sortedCitiesDictByPopulation);
+            var totalPopulationFromAllCities = CountAllPopulation(citiesDictionary);
+            Console.WriteLine($"Total population from all cities: {totalPopulationFromAllCities}");
+        }
+
+        /* Task 1. Добавьте новое значение, ключом которого является ваше имя, а значение — ваш возраст.
+         * Сделайте это с помощью метода Add. Затем добавьте еще одно значение в словарь, используя нотацию индекса.
+         * На этот раз используйте другое имя и другой возраст. Наконец, прочитайте первый элемент, который вы добавили
+         * в словарь, и запишите его в консоль с помощью Console.WriteLine.*/
+        public static void AddValuesToDictAndOutputThem()
+        {
+            Dictionary<string, int> dictionary = new Dictionary<string, int>();
+            dictionary.Add("Alina", 26);
+            dictionary["Kate"] = 29;
+            try
+            {
+                var search = dictionary.Where(p => p.Key.Contains("Alina"));
```

Comment:
I think it means the first element of the collection without any filtering

Reply:
😞 Fixed.